<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Boton_carga">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar Señal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boton_guarda">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar Señal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atrassenal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>490</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adelantesenal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>520</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rango de Tiempo</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Seleccioncanal">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>550</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Canal de Visualización</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boton_filtrar">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>500</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrar señal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boton_graficar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar canal</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graficador">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>761</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>470</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ponderacion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo Lambda</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>550</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de Umbral</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboPonderar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>470</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>One</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Single Level Noise</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboLambda">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>510</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Universal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MINIMAX</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboUmbral">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>550</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Duro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Suave</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>10</y>
      <width>671</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bookman Old Style</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>                                  Sistema básico para la visualización de señales</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
